pkg/sql/logictest/testdata/logic_test/alter_primary_key

```diff
@@ -726,14 +726,14 @@ t  CREATE TABLE public.t (
 statement ok
 DROP TABLE IF EXISTS t;
 CREATE TABLE t (x INT NOT NULL);
-ALTER TABLE t ADD CONSTRAINT "primary" PRIMARY KEY (x)
+ALTER TABLE t ADD CONSTRAINT "my_pk" PRIMARY KEY (x)
 
```

Comment:
can you also add a test to make sure "name already exists" error is returned correctly?